src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -257,6 +279,14 @@ def _try_fetch(self) -> Iterable[ExternalLocation]:
         for row in self._fetch(f"SELECT * FROM {escape_sql_identifier(self.full_name)}"):
             yield ExternalLocation(*row)
 
+    def snapshot(self, *, force_refresh: bool = False) -> list[ExternalLocation]:
```

Comment:
This is confusing as it would be the only crawler for which we do dynamically add a item when retrieving the snapshot. Furthermore, it changes implications too: we now always call the API, also when the external locations inventory table is already populated